lib/ProductOpener/Config_obf.pm

```diff
@@ -182,7 +182,7 @@ $flavor = 'obf';
 		"https://play.google.com/store/apps/details?id=org.openfoodfacts.scanner&utm_source=obf&utf_medium=web",
 	ios_app_link => "https://apps.apple.com/app/open-beauty-facts/id1122926380?utm_source=obf&utf_medium=web",
 	facebook_page_url => "https://www.facebook.com/openfoodfacts?utm_source=obf&utf_medium=web",
-	twitter_account => "OpenFoodFacts",
+	X_account => "https://x.com/OpenFoodFacts",
```

Comment:
Why add the domain name here? this variable is then used to compute the URL, or to show something like @username

Reply:
also for variables / keys, we do lowercase, so it should be x_account